src/properties.jl

```diff
@@ -253,13 +253,10 @@ function class_setproperty!(::Type{ObjectCreateProperties}, p::Properties, name:
     class_setproperty!(superclass(ObjectCreateProperties), p, name, val)
 end
 
-function get_track_order(p::Properties)
-    attr = Ref{UInt32}()
-    API.h5p_get_attr_creation_order(p, attr)
-    link = Ref{UInt32}()
-    API.h5p_get_link_creation_order(p, link)
-    attr[] != 0 && link[] != 0
-end
+get_track_order(p::Properties) = (
+    API.h5p_get_link_creation_order(p) != 0 &&
+    API.h5p_get_attr_creation_order(p) != 0
+)
```

Comment:
```julia
get_track_order(p::Properties) = API.h5p_get_link_creation_order(p) != 0 && API.h5p_get_attr_creation_order(p) != 0
```
If this is going to be a short form function, you might as well make it a one-liner  :)

Reply:
I thought `HDF5.jl` tried to adhere to some kind of `linewidth` code style convention, but as you wish ;)

Reply:
We are not that strict, usually we aim for readability instead of a hardcore line length limit

Reply:
I would be nitpicking by saying that the multi-line version is more readable (with the `!=` alignment) ;)